adder.js

```diff
@@ -1 +1,15 @@
-// code here
+function add(num1, num2) {
+  // num1 = parseInt(prompt("something"));
+  // num2 = parseInt(prompt("something"));
+  return num1 + num2;
+}
+function print(num) {
```

Comment:
technically correct, but you can use this function also to print `strings`. So maybe _message_ parameter would be more appropriate.